### ISSUE TITLE
Title Page (HTML and PDF): Year-month not same size and color as the document number

### ISSUE THREAD

**Issue by ERyan71258** [SMPTE/html-pub#235]
In the HTML and PDF, the year-month is not the same size and color as the document number:
![image](https://github.com/SMPTE/st429-19/assets/118845840/7af0f85c-226f-44a0-8fa6-18655aea4991)

Are we also going to start including the month, as well (as in "2023-09")? Month is now also included in the document headers. No issue with it; just need to know if I should start checking that the month is included.

**Comment by palemieux** [comment]
Yes, the month is now included at all times. See https://doc.smpte-doc.org/ag-02/main/#sec-document-numbering-overview.

I think the gray makes the document number (which is the most important piece of information) more readable.

**Comment by SteveLLamb** [comment]
I agree with the difference in color makes it more obvious. @ERyan71258, thoughts, maybe just make same size, but color ok?

**Comment by ERyan71258** [comment]
Make the same size. OK with the color.

Pertaining to the revision information, it does not contain the month. For the most part, it is not an issue. In a standard such as ST 2123, which is supposed to be updated every six months, there could be two releases within the same year. In that instance, could the month be added to the revision year?

**Comment by SteveLLamb** [comment]
> Pertaining to the revision information, it does not contain the month. For the most part, it is not an issue. In a standard such as ST 2123, which is supposed to be updated every six months, there could be two releases within the same year. In that instance, could the month be added to the revision year?

`pubRevisionOf` as defined in https://doc.smpte-doc.org/html-pub/main/#sec-pubRevisionOf is a string, so that is on the editor to put in the proper info, including the month pub date. See https://github.com/SMPTE/st430-1-private/blob/22f751a2d729c965aaa11e60e9f1eef4da58cba0/doc/main.html#L12 as an example, as that time we didnt' have month as default on.

**Comment by SteveLLamb** [comment]
Related to PDF issues, possible drop as per https://github.com/SMPTE/html-pub/issues/407.
